services/ui-src/src/components/accordions/InstructionsAccordion.tsx

```diff
@@ -0,0 +1,72 @@
+import { Accordion, Box, ListItem, UnorderedList } from "@chakra-ui/react";
+import { AccordionItem } from "components";
+import { AnyObject } from "types";
+import { parseCustomHtml, sanitizeAndParseHtml } from "utils";
+
+export const InstructionsAccordion = ({ verbiage, ...props }: Props) => {
+  const { buttonLabel, intro, list, text } = verbiage;
+  return (
+    <Accordion allowToggle={true} allowMultiple={true} sx={sx.root} {...props}>
+      <AccordionItem label={buttonLabel} sx={sx.item}>
+        <Box sx={sx.textBox}>{parseCustomHtml(intro)}</Box>
+        {list && parseList(list)}
+        {text && <Box sx={sx.textBox}>{parseCustomHtml(text)}</Box>}
+      </AccordionItem>
+    </Accordion>
+  );
+};
+
+export const parseList = (list: any) => {
```

Comment:
Thought: the strictest way to type this parameter (instead of `any`) would be with a recursive type, like this:
```ts
type NestedStringArray = (string | NestedStringArray[])[];
export const parseList = (list: NestedStringArray) => { ... }
```
And in fact, I doubt we would ever (or _should_ ever) have a triply-nested unordered list inside of an accordion, so we could probably get away with `(string | string[])[]`.

But, once again the `any` isn't actively hurting anybody, so there's no rush to change it in the context of this PR.